Add Node promotion zone label support (#201)
harvester/docs: docs/install/iso-install.md

1. Mount the Harvester ISO disk and boot the server by selecting the `Harvester Installer` option.
   ![iso-install.png](/img/v1.1/install/iso-install.png)
1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
      - Note: By default, the first node will be the management node of the cluster. When there are 3 nodes, the other 2 nodes added first are automatically promoted to management nodes to form an HA cluster

>       If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config by providing a URL of [Harvester configuration](./harvester-configuration.md) on the `customize the host` step. In this case, at least three different zones are required.

Reviewer: I don't think we need the tilde. 

```suggestion
      If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config by providing a URL of [Harvester configuration](./harvester-configuration.md) on the customize the host step. In this case, at least three different zones are required.
```

>       If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config via providing a URL of [Harvester configuration](./harvester-configuration.md) on the `customize the host` step. In this case, at least three different zones are required.

Reviewer: ```suggestion
      If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config by providing a URL of [Harvester configuration](./harvester-configuration.md) on the `customize the host` step. In this case, at least three different zones are required.
```

>       If you want to make sure that the node promotion mechanism will only promote compute nodes in zones different from existing management nodes to management, in the later `customize the host` step, you can provide a URL of [Harvester configuration](./harvester-configuration.md) which specify node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) of first node and other compute nodes that you want to be promoted to management node. In this case, at least three different zones are required.

Reviewer 2: ```suggestion
      If you want to make sure that only compute nodes in zones different from the existing management nodes can be promoted, in the later `customize the host` step, provide a URL of [Harvester configuration](./harvester-configuration.md) that specifies the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) of the first node and other compute nodes that you want to promote. In this case, at least three different zones are required.
```

Reviewer 2: Not sure if I understand it correctly. Please double-check~
I deleted some info for readability, and I think they should still understandable in this context

Author: That's what I'm trying to say. Good Suggestion. Thanks a lot!
